_posts/0000-01-02-cmhume.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
#3 "The time is always right to do what is right."
--Dr. Martin Luther King Jr.